client/app/utils/customTransformErrors.ts

```diff
@@ -0,0 +1,17 @@
+import { RJSFValidationError } from "@rjsf/utils";
+
+export const customTransformErrors = (
+  errors: RJSFValidationError[],
+  customFormatsErrorMessages: { [key: string]: string },
+) => {
+  return errors.map((error) => {
+    if (error.name !== "format") return error;
+    if (!customFormatsErrorMessages[error.params.format]) return error;
+    return {
+      ...error,
+      message: customFormatsErrorMessages[error.params.format],
+      // rjsf appears to display error.stack, not error.message
```

Comment:
If #316 uses error.message to show the messages, then we can use message instead of stack

Reply:
(#316)[https://github.com/bcgov/cas-registration/pull/412/files] is using `error.message` thanks!